instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/aws.go

```diff
@@ -34,8 +35,11 @@ const (
 
 type spanTimestampKey struct{}
 
+type attributesetter func(ctx context.Context, in middleware.InitializeInput) []attribute.KeyValue
```

Comment:
```suggestion
type AttributeSetter func(context.Context, middleware.InitializeInput) []attribute.KeyValue
```

This should probably be exported so that users can reference the type as well.  There is no need to name the attributes on a type definition like this, but it probably doesn't hurt either.